src/08-accessing.js

```diff
@@ -148,7 +164,9 @@ function getCityAndState(person) {}
  *  getFullAddress(person);
  *  //> "697 Pine Drive 2A, Staten Island, NY, 10306"
  */
-function getFullAddress(person) {}
+function getFullAddress(person) {
+  return `${person.address.street} ${person.address.unit}, ${person.address.city}, ${person.address.state}, ${person.address.zip}`;
```

Comment:
Remember when we have long strings in a template literal we can and should move them to new lines.  Let's knock these down to a new line.